web_autoflip_demo/autoflip_demo_UI/src/download.ts

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright 2020 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  cropWindowStorage,
+  videoBuffer,
+  audio,
+  updateAudio,
+  outputStorage,
+} from './globals';
+
+import {
+  videoPreview,
+  canvas,
+  videoRecord,
+  card2,
+  card4,
+  timerDisplay,
+} from './globals_dom';
+
+import { ffmpegWorkerCombine, ffmpegWorkerAudio } from './globals_worker';
+export function createDownload(inputWidth: number, inputHeight: number): void {
```

Comment:
Add js doc describing what the function does

Reply:
fixed.